ynput/ayon-core
# Bugfix/houdini ignore inputs on rendering

## Changelog Description
Ignore inputs on rendering.
It was brought from https://github.com/ynput/ayon-core/pull/32 as it was the valuable thing to merge in that PR.

## Testing notes:
1. Create a publish instance in Houdini
2. Add any rop node to its inputs (it should be ignored)
